docs/microbit_micropython_api.rst

```diff
@@ -102,9 +141,27 @@ Each of these pins are instances of the ``MicroBitPin`` class, which offers the
     # sets the period of the PWM output of the pin in microseconds
     # (see https://en.wikipedia.org/wiki/Pulse-width_modulation)
     pin.set_analog_period_microseconds(int)
-    # returns boolean
+    # Only available for touch pins 0, 1, and 2. Returns boolean if the pin
+    # is touched
     pin.is_touched()
 
+Except in the case of the pins marked **V2**, which offers the following API::
+    
+pin_logo::
+
+    # returns boolean for logo touch pin
+    pin_logo.is_touched()
+
+pin_speaker:
+    
+As above `MicroBitPin` class, but does not include pin.is_touched() and
```

Comment:
```suggestion
As above `MicroBitPin` class, but does not include `pin.is_touched()` and
```

Reply:
I've updated to `As above ``MicroBitPin`` class, but does not include ``pin.is_touched()`` and ` in  https://github.com/bbcmicrobit/micropython/pull/692/commits/f5faefa42a44db470fd018c5470232c80a1d4284